### PR TITLE
Rename BPTC and RGTC extensions to EXT_.

### PR DESCRIPTION
**Pull request by jdashg**
Also retarget BPTC from the ARB_ ext to the EXT_ one, since EXT_ is the
ES version.

CC @Oletus, how does this sound?

### CODE REVIEW

**Comment by kenrussell** [comment]
Oops, sorry, I merged Olli's pull request before this one. @jdashg can you rebase?

**Comment by jdashg** [comment]
@kenrussell Rebased.

**Comment by jdashg** [comment]
Since we're exposing the native extensions directly, I think we should use the native (ideally GLES) extension names.

**Comment by jdashg** [comment]
@Oletus PTAL

File: extensions/EXT_compressed_texture_bptc/extension.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 
-<draft href="WEBGL_compressed_texture_bptc/">
-  <name>WEBGL_compressed_texture_bptc</name>
+<draft href="EXT_compressed_texture_bptc/">
+  <name>EXT_compressed_texture_bptc</name>
```
[the remark this answers is not in the document]
Comment (jdashg) [reply extensions/EXT_compressed_texture_bptc/extension.xml]: Fixed.

File: extensions/EXT_compressed_texture_rgtc/extension.xml [hunk 2]
```diff
@@ -119,5 +119,8 @@ interface WEBGL_compressed_texture_rgtc {
     <revision date="2018/09/26">
       <change>Initial revision.</change>
     </revision>
+    <revision date="2018/09/26">
+      <change>Rename to EXT_texture_compression_rgtc from WEBGL_texture_compression_rgtc</change>
```
[the remark this answers is not in the document]
Comment (jdashg) [reply extensions/EXT_compressed_texture_rgtc/extension.xml]: Fixed.

File: extensions/EXT_compressed_texture_rgtc/extension.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 
-<draft href="WEBGL_compressed_texture_rgtc/">
-  <name>WEBGL_compressed_texture_rgtc</name>
+<draft href="EXT_compressed_texture_rgtc/">
+  <name>EXT_compressed_texture_rgtc</name>
```
[the remark this answers is not in the document]
Comment (jdashg) [reply extensions/EXT_compressed_texture_rgtc/extension.xml]: I prefer using the native ones where we don't have any real caveats compared to the native ext.

File: extensions/EXT_texture_compression_rgtc/extension.xml [hunk 4]
```diff
@@ -64,7 +64,7 @@
   </overview>
   <idl xml:space="preserve">
 [NoInterfaceObject]
-interface WEBGL_compressed_texture_rgtc {
+interface EXT_compressed_texture_rgtc {
```
Comment (Oletus) [inline extensions/EXT_texture_compression_rgtc/extension.xml]: The name is still incorrect here.

**Comment by jdashg** [comment]
@Oletus oops, ptal again

**Comment by kenrussell** [comment]
LGTM after the fact.

**Pull request merged**